sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -127,33 +127,35 @@ public abstract partial class NullableResponse<T>
         public abstract Azure.Response GetRawResponse();
         public override string ToString() { throw null; }
     }
-    public abstract partial class Operation
+    public partial class Operation
```

Comment:
One suggestion is to add these APIs to an internal subtype of `Operation` and return that anywhere this instance is needed.  I believe we would prefer to keep the `Operation` type itself internal, as that lets it evolve better in the future, helps us address performance issues, etc.

Reply:
Sorry, did not add enough explanation for this.
The goal is to implement LRO rehydration for both DPG and MPG. 
Had a discussion with @m-nash , @KrzysztofCwalina and @tg-msft, given the way we are implementing LRO for DPG nowadays, the way we would like to implement it is to make Operation non-abstract.

Reply:
I think we have two choices: 1) we could make the type non-abstract as in this PR, 2) we could add a static method Operation.Rehydrate that returns an internal subtype of Operation. I know that we initially said that we wanted a ctor, but now that I think about it the static method could be better. Rehydration is an advanced scenario and the static method would hide the API a bit. Also, it would allow us to not have to add fields to Operation

Reply:
OK. Static method is better to me as well. Given we don't have a good way to get the generic type from constructor directly, we are using reflection to check the constructor of the generic type in MPG to distinguish Resource and Model.
I will create another PR to implement it with static method to not mess with the existing change.

Reply:
Created https://github.com/Azure/azure-sdk-for-net/pull/42686 for static methods